packages/react-native/Libraries/AppDelegate/RCTRootViewFactory.mm

```diff
@@ -266,7 +259,7 @@ - (RCTHost *)createReactHost:(NSDictionary *)launchOptions
   __weak __typeof(self) weakSelf = self;
   RCTHost *reactHost =
       [[RCTHost alloc] initWithBundleURLProvider:self->_configuration.bundleURLBlock
-                                    hostDelegate:self
+                                    hostDelegate:_hostDelegate
```

Comment:
So if we don't provide hostDelegate is everything going to work fine as before?

Reply:
I would think `initWithConfiguration: andTurboModuleManagerDelegate` should be deprecated. The `RCTAppDelegate` is the hostDelegate and should always be passed.

Reply:
Makes sense